### ISSUE TITLE
Archive authentication did not verify : Prevent use about TAM correction to apply

### ISSUE THREAD

**Issue by camlafit** [borgbase/vorta#1937]
### Description

Hi

Vorta can't finish any backup or list archive with this error. 
Looks related to  : 
https://borgbackup.readthedocs.io/en/stable/changes.html#pre-1-2-5-archives-spoofing-vulnerability-cve-2023-36811

Could be interesting to target to official borgbackup information to repair this.

As effect, without correction, vorta loop if option "run missed backups on startup or wake up" is enable.

As borg > 1.2.6 is now in some distributions as debian backports, error could be raised without useful notice

### Reproduction

- [X] I tried to reproduce the issue.
- [ ] I was able to reproduce the issue.

### OS

Debian 12

### Version of Vorta

0.8.10-1+deb12u1, 0.9.1~beta2-1~exp1

### What did you install Vorta with?

Distribution package

### Version of Borg

1.2.7-2~bpo12+1

### Logs

```pytb
2024-02-14 14:45:58,810 - vorta.borg.borg_job - ERROR - Data integrity error: Archive authentication did not verify
2024-02-14 14:45:58,810 - vorta.borg.borg_job - ERROR - Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/borg/archiver.py", line 5343, in main
    exit_code = archiver.run(args)
                ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/archiver.py", line 5263, in run
    return set_ec(func(args))
                  ^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/archiver.py", line 189, in wrapper
    return method(self, args, repository=repository, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/archiver.py", line 1195, in do_delete
    return self._delete_archives(args, repository)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/archiver.py", line 1243, in _delete_archives
    with Cache(repository, key, manifest, progress=args.progress, lock_wait=self.lock_wait, iec=args.iec) as cache:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/cache.py", line 387, in __new__
    return local()
           ^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/cache.py", line 378, in local
    return LocalCache(repository=repository, key=key, manifest=manifest, path=path, sync=sync,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/cache.py", line 497, in __init__
    self.sync()
  File "/usr/lib/python3/dist-packages/borg/cache.py", line 904, in sync
    self.chunks = create_master_idx(self.chunks)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/cache.py", line 858, in create_master_idx
    fetch_and_build_idx(archive_id, decrypted_repository, archive_chunk_idx)
  File "/usr/lib/python3/dist-packages/borg/cache.py", line 758, in fetch_and_build_idx
    archive, verified, _ = self.key.unpack_and_verify_archive(data, force_tam_not_required=True)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/borg/crypto/key.py", line 329, in unpack_and_verify_archive
    raise ArchiveTAMInvalid()
borg.crypto.key.ArchiveTAMInvalid: Data integrity error: Archive authentication did not verify

Platform: Linux 6.5.0-0.deb12.4-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.5.10-1~bpo12+1 (2023-11-23) x86_64
Linux: Unknown Linux  
Borg: 1.2.7  Python: CPython 3.11.2 msgpack: 1.0.3 fuse: pyfuse3 3.2.1 [pyfuse3,llfuse]
PID: 235349  CWD: /home/user
sys.argv: ['/usr/bin/borg', 'delete', '--rsh=ssh -i ~/.ssh/id_ed25519', '--info', '--log-json', 'bckuser@bckborg:/backup/user/archive::name-2024-02-14-110603'
, 'BACKUP-2024-02-14-104835']
SSH_ORIGINAL_COMMAND: None
```

**Comment by camlafit** [comment]
Hello

Indeed it's a duplicate issue. Sorry I've searched before to open this.

In my head, could be interesting to detect ArchiveTAMInvalid error and provide at least a link to borgbackup documentation.
In my case I do a simple debian upgrade and don't catch breaking change and detect it some time later.

**Comment by m3nu** [comment]
No worries. I'll add a message dialog then.

**Comment by camlafit** [comment]
closed ad duplicate #1794
